django/middleware/csrf.py

```diff
@@ -204,7 +204,6 @@ def _accept(self, request):
         # request.  This will be relevant when both decorator and middleware
         # are used.
         request.csrf_processing_done = True
-        return None
```

Comment:
"Explicit is better than implicit" … although… there are many cases in Django where implicit None is returned.

In any case it's not worth the change imho 🤷‍♂️